spec/unit/parser/functions/fqdn_rand_spec.rb

```diff
@@ -38,6 +38,20 @@
     expect(val1).not_to eql(val2)
   end
 
+  it "should return a specific value with given set of inputs on non-fips enabled host" do
+    Puppet::Util::Platform.stubs(:fips_enabled?).returns false
+    val = fqdn_rand(3000, :host => 'dummy.fqdn.net')
```

Comment:
Could you also include an explicit seed value too?

Reply:
I fixed the nits (tightened) that I will push. But trying to do something like 'expect(fqdn_rand(3000, 'expensive job 33')).to eql(338)' generates errors. Also not sure if we really can supply a seed externally - I understand all inputs to fqdn_rand than max go into forming the seed but wanted to know what you were looking for.

Reply:
Sorry, the test has two `fqdn_rand`s, the test helper and the method on the scope object. I think you can accomplish specifying both the fqdn and seed by specifying `:extra_identifier => ['expensive job 33']`.